pwa/app/routes/events.($year).tsx

```diff
@@ -26,6 +27,12 @@ async function loadData(params: Params) {
     });
   }
 
+  if (events.data.length === 0) {
```

Comment:
Maybe the API should be 404ing instead of returning zero-length arrays but oh well